resources/scripts/components/server/features/PIDLimitModalFeature.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useState } from 'react';
+import { ServerContext } from '@/state/server';
+import Modal from '@/components/elements/Modal';
+import tw from 'twin.macro';
+import Button from '@/components/elements/Button';
+import FlashMessageRender from '@/components/FlashMessageRender';
+import useFlash from '@/plugins/useFlash';
+import { SocketEvent } from '@/components/server/events';
+import { useStoreState } from 'easy-peasy';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faExclamationTriangle } from '@fortawesome/free-solid-svg-icons';
+
+const PIDLimitModalFeature = () => {
+    const [ visible, setVisible ] = useState(false);
+    const [ loading ] = useState(false);
+
+    const status = ServerContext.useStoreState(state => state.status.value);
+    const { clearFlashes } = useFlash();
+    const { connected, instance } = ServerContext.useStoreState(state => state.socket);
+    const isAdmin = useStoreState(state => state.user.data!.rootAdmin);
+
+    useEffect(() => {
+        if (!connected || !instance || status === 'running') return;
+
+        const errors = [
+            'pthread_create failed',
+            'Exception in thread "Craft Async Scheduler Management Thread"',
+            'unable to create new native thread',
+            'unable to create native thread',
+        ];
+
+        const listener = (line: string) => {
+            if (errors.some(p => line.toLowerCase().includes(p))) {
+                setVisible(true);
+            }
+        };
+
+        instance.addListener(SocketEvent.CONSOLE_OUTPUT, listener);
+
+        return () => {
+            instance.removeListener(SocketEvent.CONSOLE_OUTPUT, listener);
+        };
+    }, [ connected, instance, status ]);
+
+    useEffect(() => {
+        clearFlashes('feature:pidLimit');
+    }, []);
+
+    return (
+        <Modal visible={visible} onDismissed={() => setVisible(false)} closeOnBackground={false} showSpinnerOverlay={loading}>
+            <FlashMessageRender key={'feature:pidLimit'} css={tw`mb-4`} />
+            {isAdmin ?
+                <>
+                    <div css={tw`mt-4 sm:flex items-center`}>
+                        <FontAwesomeIcon css={tw`pr-4`} icon={faExclamationTriangle} color={'orange'} size={'4x'}/>
+                        <h2 css={tw`text-2xl mb-4 text-neutral-100 `}>Memory or process limit reached...</h2>
+                    </div>
+                    <p css={tw`mt-4`}>The server has reached the maximum process or memory limit.</p>
+                    <p css={tw`mt-4`}>Please increase the <code css={tw`font-mono bg-neutral-900`}>container_pid_limit</code> in the Wings <code css={tw`font-mono bg-neutral-900`}>config.yml</code> file, located at <code css={tw`font-mono bg-neutral-900`}>/etc/pterodactyl</code>, by default.</p>
```

Comment:
This message should be worded to let them know the config option exists and that increasing the value may help resolve the issue, rather than telling them to change it.

Reply:
It's not in question if it `may help resolve the issue`, It will resolve the issue, as that's the solution... Or do we just tell them to reduce the plugins / mods / whatever it maybe causing it to use more then 512 processes

Reply:
> It's not in question if it `may help resolve the issue`, It will resolve the issue, as that's the solution... Or do we just tell them to reduce the plugins / mods / whatever it maybe causing it to use more then 512 processes

That's assuming there isn't an underlying problem causing that amount of processes to be created, while you are right it will resolve the problem, it may not be something you will want to change.

Also, didn't we change the default to `1024`?

Reply:
> > It's not in question if it `may help resolve the issue`, It will resolve the issue, as that's the solution... Or do we just tell them to reduce the plugins / mods / whatever it maybe causing it to use more then 512 processes
> 
> That's assuming there isn't an underlying problem causing that amount of processes to be created, while you are right it will resolve the problem, it may not be something you will want to change.
> 
> Also, didn't we change the default to `1024`?

Default was 256 and later bumped to 512.